.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Release
+on:
+  release:
+    types:
+      - created
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - run: ./ciScripts/buildPlugin.sh
+        env:
+          RELEASE_NOTES: ${{ github.event.release.body }}
```

Comment:
One of the things we could do is create the 
Release notes, that get bundled in the plugin, by using the markdown from the GitHub release, that way the users see what's in the new release, wdyt?

Reply:
We have a link in the update notification, do you still think it also is needed in the change notes as well?

Reply:
I am appending it to the release notes.